config/api.yml

```diff
@@ -679,6 +679,8 @@
         :identifier: image_show_list
       - :name: set_ownership
         :identifier: image_ownership
+      - :name: import
+        :identifier: image_import
```

Comment:
https://github.com/ManageIQ/manageiq/pull/21112/files#diff-cdcd147fb85ef58d008a4ad0cfa3ddfea649382cb062407e2342aff07362b6cfR6734 has `import_image` not sure which one of these is correct

Reply:
Thanks you @h-kataria. My mistake, I've swapped the words for some reason. I started now a new cross-repo test to be sure.